static/js/around-map.js

```diff
@@ -173,7 +173,14 @@ function scrollCard (poi_id) {
   window.scrollTo(0, (cardTop - cardHeight/2))  
 }
 
-// saving/adding a spot - form
+// pois form
+const displayForm = document.querySelectorAll('.display-form')
+displayForm.forEach( button => {
```

Comment:
Still a mistery why `.display-form` is not a form but a list of clickable buttons.
Could you rename so it's more explicit?